tcms/rpc/tests/test_testcase.py

```diff
@@ -173,6 +173,9 @@ def test_filter_query_none(self):
         self.assertIn("author", result[0])
         self.assertIn("default_tester", result[0])
         self.assertIn("reviewer", result[0])
+        self.assertIn("setup_duration_ms", result[0])
+        self.assertIn("testing_duration_ms", result[0])
+        self.assertIn("expected_duration_ms", result[0])
```

Comment:
I think you should configure setup_duration & testing_duration fields in the test setup part above (145-150) to some distinctive hard-coded values and then assert that the result contains the exact same values in seconds.  This will help us figure out if the annotation works on all DB engines b/c we do test on all of them.  

Indeed there's some failure with Postgres, not sure if it is related. Let's simplify and try figuring it out.